VideoLocker/res/layout/activity_login.xml

```diff
@@ -59,47 +59,74 @@
                         android:paddingLeft="44dp"
                         android:paddingRight="44dp">
 
-                        <EditText
-                            android:id="@+id/email_et"
-                            style="@style/edit_text_style"
-                            android:hint="@string/email_username"
-                            android:inputType="textEmailAddress"
-                            android:maxLength="100" />
-
-                        <EditText
-                            android:id="@+id/password_et"
-                            style="@style/edit_text_style"
-                            android:layout_marginTop="8dp"
-                            android:hint="@string/password"
-                            android:inputType="textPassword"
-                            android:maxLength="100" />
+                        <android.support.design.widget.TextInputLayout
+                            android:id="@+id/usernameWrapper"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:paddingStart="-4dp"
+                            android:paddingEnd="-4dp"
+                            android:paddingLeft="-4dp"
+                            android:paddingRight="-4dp"
```

Comment:
@bguertin I think this is what you meant by the extra default padding. I suppose we could just apply "-4dp" here, but would it be better to just have a custom TextInputLayout that removes the extra padding? Would we want to always remove the extra padding for the register screen text inputs and future text inputs?